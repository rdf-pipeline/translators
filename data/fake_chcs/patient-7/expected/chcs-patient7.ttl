# generated on  now

@prefix Bundle: <http://hl7.org/fhir/Bundle> .
@prefix fhir: <http://hl7.org/fhir/> .
@prefix xhtml: <http://www.w3.org/1999/xhtml> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema> .
@base <http://this-fhir-server/fhir/> .

[] a fhir:Bundle;
   fhir:Bundle.type [ fhir:value "document" ];
   fhir:Bundle.entry [ 
      a fhir:Bundle.entry;
      fhir:Bundle.entry.resource [ 
         a fhir:Resource;
       ];
      fhir:Bundle.entry.resource [ 
         a fhir:Resource;
       ];
      fhir:Bundle.entry.resource [ 
         a fhir:Resource;
       ];
      fhir:Bundle.entry.resource [ 
         a fhir:Resource;
       ];
      fhir:Bundle.entry.resource [ 
         a fhir:Resource;
       ];
      fhir:Bundle.entry.resource [ 
         a fhir:Resource;
       ];
      fhir:Bundle.entry.resource [ 
         a fhir:Resource;
       ];
      fhir:Bundle.entry.resource [ 
         a fhir:Resource;
       ];
      fhir:Bundle.entry.resource [ 
         a fhir:Resource;
       ];
      fhir:Bundle.entry.resource [ 
         a fhir:Resource;
       ];
      fhir:Bundle.entry.resource [ 
         a fhir:Resource;
       ];
    ];
 .
<Patient/2-000007> fhir:Reference.display [ fhir:value "BUNNY,BUGS" ] .
<6-111111> fhir:Reference.display [ fhir:value "DUCK,DONALD" ] .
<101-227995> fhir:Reference.display [ fhir:value "941103-00522" ] .
<59_2-1> fhir:Reference.display [ fhir:value "MAIN PHARMACY" ] .
<Patient/2-000007> fhir:Reference.display [ fhir:value "BUNNY,BUGS" ] .
<6-11111> fhir:Reference.display [ fhir:value "DUCK,DONALD" ] .
<101-16145150> fhir:Reference.display [ fhir:value "090420-03671" ] .
<59_2-12> fhir:Reference.display [ fhir:value "ANY PHARMACY (SOME MED)" ] .
<Patient/2-000007> fhir:Reference.display [ fhir:value "BUNNY, BUGS" ] .
<6-11111> fhir:Reference.display [ fhir:value "DUCK,DONALD" ] .
<101-16145151> fhir:Reference.display [ fhir:value "090420-03672" ] .
<59_2-12> fhir:Reference.display [ fhir:value "ANY PHARMACY (SOME MED)" ] .
<Patient/2-000007> fhir:Reference.display [ fhir:value "BUNNY,BUGS" ] .
<6-111111> fhir:Reference.display [ fhir:value "DUCK,DONALD" ] .
<101-227995> fhir:Reference.display [ fhir:value "941103-00522" ] .
<59_2-1> fhir:Reference.display [ fhir:value "MAIN PHARMACY" ] .
<Patient/2-000007> fhir:Reference.display [ fhir:value "BUNNY,BUGS" ] .
<6-11111> fhir:Reference.display [ fhir:value "DUCK,DONALD" ] .
<101-16145150> fhir:Reference.display [ fhir:value "090420-03671" ] .
<59_2-12> fhir:Reference.display [ fhir:value "ANY PHARMACY (SOME MED)" ] .
<Patient/2-000007> fhir:Reference.display [ fhir:value "BUNNY, BUGS" ] .
<6-11111> fhir:Reference.display [ fhir:value "DUCK,DONALD" ] .
<101-16145151> fhir:Reference.display [ fhir:value "090420-03672" ] .
<59_2-12> fhir:Reference.display [ fhir:value "ANY PHARMACY (SOME MED)" ] .
<Patient/2-000007> fhir:Reference.display [ fhir:value "BUNNY,BUGS" ] .
<Patient/2-000007> fhir:Reference.display [ fhir:value "BUNNY,BUGS" ] .
<Patient/2-000007> fhir:Reference.display [ fhir:value "BUNNY,BUGS" ] .
<Patient/2-000007> fhir:Reference.display [ fhir:value "BUNNY, BUGS" ] .
