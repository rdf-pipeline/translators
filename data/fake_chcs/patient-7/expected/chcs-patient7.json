{
  "resourceType": "Bundle",
  "type": "document",
  "entry": [
    {
      "resourceType": "Composition",
      "date": "now",
      "status": "final",
      "type": {
        "coding": [
          {
            "system": "chcs",
            "version": "0.0.1",
            "code": "chcs",
            "display": "chcs",
            "userSelected": false
          }
        ],
        "text": "chcs"
      },
      "resource": [
        {
          "resourceType": "Patient",
          "id": "urn:local:fhir:Patient:2-000007",
          "identifier": [
            {
              "use": "usual",
              "type": {
                "coding": [
                  {
                    "code": "chcss",
                    "display": "chcss"
                  }
                ],
                "text": "777777777"
              },
              "value": "777777777"
            }
          ],
          "name": {
            "use": "usual",
            "family": [
              "BUNNY"
            ],
            "given": [
              "BUGS DOC"
            ]
          },
          "gender": "male",
          "birthDate": "01-01-1990",
          "address": [
            {
              "resourceType": "Address",
              "type": "postal",
              "line": [
                "100 MAIN ST"
              ],
              "city": "ANYTOWN"
            }
          ],
          "maritalStatus": {
            "coding": [
              {
                "system": "http://hl7.org/fhir/marital-status",
                "code": "D",
                "display": "D"
              }
            ],
            "text": "D"
          }
        },
        {
          "resourceType": "MedicationDispense",
          "id": "urn:local:fhir:MedicationDispense:52-40863",
          "identifier": {
            "use": "usual",
            "type": {
              "coding": [
                {
                  "system": "chcs",
                  "version": "0.0.1",
                  "code": "chcs",
                  "display": "Prescription",
                  "userSelected": false
                }
              ],
              "text": "Prescription"
            },
            "system": "chcs",
            "value": "52-40863"
          },
          "status": "completed",
          "medicationCodeableConcept": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "PAINKILLER 400MG TAB",
                "userSelected": false
              }
            ],
            "text": "PAINKILLER 400MG TAB"
          },
          "medicationReference": {
            "reference": "50-260",
            "display": "PAINKILLER 400MG TAB"
          },
          "patient": {
            "reference": "Patient/2-000007",
            "display": "BUNNY,BUGS"
          },
          "dispenser": {
            "reference": "6-111111",
            "display": "DUCK,DONALD"
          },
          "authorizingPrescription": [
            {
              "reference": "101-227995",
              "display": "941103-00522"
            }
          ],
          "quantity": {
            "value": 120,
            "unit": "unsupplied",
            "system": "chcs",
            "code": "chcs"
          },
          "daysSupply": {
            "value": 120,
            "unit": "unsupplied",
            "system": "chcs",
            "code": "chcs"
          },
          "whenPrepared": "1990-01-01T00:00:00Z",
          "whenHandedOver": "1990-01-01T00:00:00Z",
          "destination": {
            "reference": "59_2-1",
            "display": "MAIN PHARMACY"
          },
          "note": [
            {
              "text": "NONE"
            }
          ],
          "dosageInstruction": [
            {
              "text": "T1 TAB Q1D #120 RF0",
              "timing": {
                "resourceType": "Timing",
                "event": [
                  "1990-01-01T00:00:00Z"
                ],
                "repeat": {
                  "frequency": 1,
                  "period": 1,
                  "periodUnit": "d"
                },
                "code": {
                  "coding": [
                    {
                      "system": "chcs",
                      "version": "0.0.1",
                      "code": "chcs",
                      "display": "T1 TAB Q1D #120 RF0",
                      "userSelected": false
                    }
                  ],
                  "text": "T1 TAB Q1D #120 RF0"
                }
              },
              "asNeededBoolean": false
            }
          ]
        },
        {
          "resourceType": "MedicationDispense",
          "id": "urn:local:fhir:MedicationDispense:52-7810413",
          "identifier": {
            "use": "usual",
            "type": {
              "coding": [
                {
                  "system": "chcs",
                  "version": "0.0.1",
                  "code": "chcs",
                  "display": "Prescription",
                  "userSelected": false
                }
              ],
              "text": "Prescription"
            },
            "system": "chcs",
            "value": "52-7810413"
          },
          "status": "completed",
          "medicationCodeableConcept": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "OYS SHL CALCIUM 500MG + VIT D 200IU",
                "userSelected": false
              }
            ],
            "text": "OYS SHL CALCIUM 500MG + VIT D 200IU"
          },
          "medicationReference": {
            "reference": "50-234072",
            "display": "OYS SHL CALCIUM 500MG + VIT D 200IU"
          },
          "patient": {
            "reference": "Patient/2-000007",
            "display": "BUNNY,BUGS"
          },
          "dispenser": {
            "reference": "6-11111",
            "display": "DUCK,DONALD"
          },
          "authorizingPrescription": [
            {
              "reference": "101-16145150",
              "display": "090420-03671"
            }
          ],
          "quantity": {
            "value": 60,
            "unit": "unsupplied",
            "system": "chcs",
            "code": "chcs"
          },
          "daysSupply": {
            "value": 30,
            "unit": "unsupplied",
            "system": "chcs",
            "code": "chcs"
          },
          "whenPrepared": "1990-01-01T00:00:00",
          "whenHandedOver": "1990-01-01T00:00:00",
          "destination": {
            "reference": "59_2-12",
            "display": "ANY PHARMACY (SOME MED)"
          },
          "note": [
            {
              "text": "215/14"
            }
          ],
          "dosageInstruction": [
            {
              "text": "T1 TB PO BID #60 RF3",
              "additionalInstructions": {
                "coding": [
                  {
                    "system": "chcs",
                    "version": "0.0.1",
                    "code": "chcs",
                    "display": "TAKE ONE TABLET BY MOUTH~TWICE A DAY  ~",
                    "userSelected": false
                  }
                ],
                "text": "TAKE ONE TABLET BY MOUTH~TWICE A DAY  ~"
              },
              "timing": {
                "resourceType": "Timing",
                "event": [
                  "1990-01-01T00:00:00"
                ],
                "repeat": {
                  "frequency": 2,
                  "period": 1,
                  "periodUnit": "d"
                },
                "code": {
                  "coding": [
                    {
                      "system": "chcs",
                      "version": "0.0.1",
                      "code": "chcs",
                      "display": "T1 TB PO BID #60 RF3",
                      "userSelected": false
                    }
                  ],
                  "text": "T1 TB PO BID #60 RF3"
                }
              },
              "asNeededBoolean": false
            }
          ]
        },
        {
          "resourceType": "MedicationDispense",
          "id": "urn:local:fhir:MedicationDispense:52-7810414",
          "identifier": {
            "use": "usual",
            "type": {
              "coding": [
                {
                  "system": "chcs",
                  "version": "0.0.1",
                  "code": "chcs",
                  "display": "Prescription",
                  "userSelected": false
                }
              ],
              "text": "Prescription"
            },
            "system": "chcs",
            "value": "52-7810414"
          },
          "status": "completed",
          "medicationCodeableConcept": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "RECLIPSEN TAB 28-DAY (DESOGEN EQ)",
                "userSelected": false
              }
            ],
            "text": "RECLIPSEN TAB 28-DAY (DESOGEN EQ)"
          },
          "medicationReference": {
            "reference": "50-3621",
            "display": "RECLIPSEN TAB 28-DAY (DESOGEN EQ)"
          },
          "patient": {
            "reference": "Patient/2-000007",
            "display": "BUNNY, BUGS"
          },
          "dispenser": {
            "reference": "6-11111",
            "display": "DUCK,DONALD"
          },
          "authorizingPrescription": [
            {
              "reference": "101-16145151",
              "display": "090420-03672"
            }
          ],
          "quantity": {
            "value": 169,
            "unit": "unsupplied",
            "system": "chcs",
            "code": "chcs"
          },
          "daysSupply": {
            "value": 84,
            "unit": "unsupplied",
            "system": "chcs",
            "code": "chcs"
          },
          "whenPrepared": "1990-01-01T00:00:00",
          "whenHandedOver": "1990-01-01T00:00:00",
          "destination": {
            "reference": "59_2-12",
            "display": "ANY PHARMACY (SOME MED)"
          },
          "note": [
            {
              "text": "215/14"
            }
          ],
          "dosageInstruction": [
            {
              "text": "T1 TAB PO QD #169 RF0",
              "additionalInstructions": {
                "coding": [
                  {
                    "system": "chcs",
                    "version": "0.0.1",
                    "code": "chcs",
                    "display": "TAKE ONE TABLET BY MOUTH~EVERY DAY  ~",
                    "userSelected": false
                  }
                ],
                "text": "TAKE ONE TABLET BY MOUTH~EVERY DAY  ~"
              },
              "timing": {
                "resourceType": "Timing",
                "event": [
                  "1990-01-01T00:00:00"
                ],
                "code": {
                  "coding": [
                    {
                      "system": "chcs",
                      "version": "0.0.1",
                      "code": "chcs",
                      "display": "T1 TAB PO QD #169 RF0",
                      "userSelected": false
                    }
                  ],
                  "text": "T1 TAB PO QD #169 RF0"
                }
              },
              "asNeededBoolean": false
            }
          ]
        },
        {
          "resourceType": "MedicationDispense",
          "id": "urn:local:fhir:MedicationDispense:52-40863",
          "identifier": {
            "use": "usual",
            "type": {
              "coding": [
                {
                  "system": "chcs",
                  "version": "0.0.1",
                  "code": "chcs",
                  "display": "Prescription",
                  "userSelected": false
                }
              ],
              "text": "Prescription"
            },
            "system": "chcs",
            "value": "52-40863"
          },
          "status": "completed",
          "medicationCodeableConcept": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "PAINKILLER 400MG TAB",
                "userSelected": false
              }
            ],
            "text": "PAINKILLER 400MG TAB"
          },
          "medicationReference": {
            "reference": "50-260",
            "display": "PAINKILLER 400MG TAB"
          },
          "patient": {
            "reference": "Patient/2-000007",
            "display": "BUNNY,BUGS"
          },
          "dispenser": {
            "reference": "6-111111",
            "display": "DUCK,DONALD"
          },
          "authorizingPrescription": [
            {
              "reference": "101-227995",
              "display": "941103-00522"
            }
          ],
          "quantity": {
            "value": 120,
            "unit": "unsupplied",
            "system": "chcs",
            "code": "chcs"
          },
          "daysSupply": {
            "value": 120,
            "unit": "unsupplied",
            "system": "chcs",
            "code": "chcs"
          },
          "whenPrepared": "1990-01-01T00:00:00",
          "whenHandedOver": "1990-01-01T00:00:00",
          "destination": {
            "reference": "59_2-1",
            "display": "MAIN PHARMACY"
          },
          "note": [
            {
              "text": "NONE"
            }
          ],
          "dosageInstruction": [
            {
              "text": "T1 TAB Q1D #120 RF0",
              "timing": {
                "resourceType": "Timing",
                "event": [
                  "1990-01-01T00:00:00"
                ],
                "repeat": {
                  "frequency": 1,
                  "period": 1,
                  "periodUnit": "d"
                },
                "code": {
                  "coding": [
                    {
                      "system": "chcs",
                      "version": "0.0.1",
                      "code": "chcs",
                      "display": "T1 TAB Q1D #120 RF0",
                      "userSelected": false
                    }
                  ],
                  "text": "T1 TAB Q1D #120 RF0"
                }
              },
              "asNeededBoolean": false
            }
          ]
        },
        {
          "resourceType": "MedicationDispense",
          "id": "urn:local:fhir:MedicationDispense:52-7810413",
          "identifier": {
            "use": "usual",
            "type": {
              "coding": [
                {
                  "system": "chcs",
                  "version": "0.0.1",
                  "code": "chcs",
                  "display": "Prescription",
                  "userSelected": false
                }
              ],
              "text": "Prescription"
            },
            "system": "chcs",
            "value": "52-7810413"
          },
          "status": "completed",
          "medicationCodeableConcept": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "OYS SHL CALCIUM 500MG + VIT D 200IU",
                "userSelected": false
              }
            ],
            "text": "OYS SHL CALCIUM 500MG + VIT D 200IU"
          },
          "medicationReference": {
            "reference": "50-234072",
            "display": "OYS SHL CALCIUM 500MG + VIT D 200IU"
          },
          "patient": {
            "reference": "Patient/2-000007",
            "display": "BUNNY,BUGS"
          },
          "dispenser": {
            "reference": "6-11111",
            "display": "DUCK,DONALD"
          },
          "authorizingPrescription": [
            {
              "reference": "101-16145150",
              "display": "090420-03671"
            }
          ],
          "quantity": {
            "value": 60,
            "unit": "unsupplied",
            "system": "chcs",
            "code": "chcs"
          },
          "daysSupply": {
            "value": 30,
            "unit": "unsupplied",
            "system": "chcs",
            "code": "chcs"
          },
          "whenPrepared": "1990-01-01T00:00:00",
          "whenHandedOver": "1990-01-01T00:00:00",
          "destination": {
            "reference": "59_2-12",
            "display": "ANY PHARMACY (SOME MED)"
          },
          "note": [
            {
              "text": "215/14"
            }
          ],
          "dosageInstruction": [
            {
              "text": "T1 TB PO BID #60 RF3",
              "additionalInstructions": {
                "coding": [
                  {
                    "system": "chcs",
                    "version": "0.0.1",
                    "code": "chcs",
                    "display": "TAKE ONE TABLET BY MOUTH~TWICE A DAY  ~",
                    "userSelected": false
                  }
                ],
                "text": "TAKE ONE TABLET BY MOUTH~TWICE A DAY  ~"
              },
              "timing": {
                "resourceType": "Timing",
                "event": [
                  "1990-01-01T00:00:00"
                ],
                "repeat": {
                  "frequency": 2,
                  "period": 1,
                  "periodUnit": "d"
                },
                "code": {
                  "coding": [
                    {
                      "system": "chcs",
                      "version": "0.0.1",
                      "code": "chcs",
                      "display": "T1 TB PO BID #60 RF3",
                      "userSelected": false
                    }
                  ],
                  "text": "T1 TB PO BID #60 RF3"
                }
              },
              "asNeededBoolean": false
            }
          ]
        },
        {
          "resourceType": "MedicationDispense",
          "id": "urn:local:fhir:MedicationDispense:52-7810414",
          "identifier": {
            "use": "usual",
            "type": {
              "coding": [
                {
                  "system": "chcs",
                  "version": "0.0.1",
                  "code": "chcs",
                  "display": "Prescription",
                  "userSelected": false
                }
              ],
              "text": "Prescription"
            },
            "system": "chcs",
            "value": "52-7810414"
          },
          "status": "completed",
          "medicationCodeableConcept": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "RECLIPSEN TAB 28-DAY (DESOGEN EQ)",
                "userSelected": false
              }
            ],
            "text": "RECLIPSEN TAB 28-DAY (DESOGEN EQ)"
          },
          "medicationReference": {
            "reference": "50-3621",
            "display": "RECLIPSEN TAB 28-DAY (DESOGEN EQ)"
          },
          "patient": {
            "reference": "Patient/2-000007",
            "display": "BUNNY, BUGS"
          },
          "dispenser": {
            "reference": "6-11111",
            "display": "DUCK,DONALD"
          },
          "authorizingPrescription": [
            {
              "reference": "101-16145151",
              "display": "090420-03672"
            }
          ],
          "quantity": {
            "value": 169,
            "unit": "unsupplied",
            "system": "chcs",
            "code": "chcs"
          },
          "daysSupply": {
            "value": 84,
            "unit": "unsupplied",
            "system": "chcs",
            "code": "chcs"
          },
          "whenPrepared": "1990-01-01T00:00:00",
          "whenHandedOver": "1990-01-01T00:00:00",
          "destination": {
            "reference": "59_2-12",
            "display": "ANY PHARMACY (SOME MED)"
          },
          "note": [
            {
              "text": "215/14"
            }
          ],
          "dosageInstruction": [
            {
              "text": "T1 TAB PO QD #169 RF0",
              "additionalInstructions": {
                "coding": [
                  {
                    "system": "chcs",
                    "version": "0.0.1",
                    "code": "chcs",
                    "display": "TAKE ONE TABLET BY MOUTH~EVERY DAY  ~",
                    "userSelected": false
                  }
                ],
                "text": "TAKE ONE TABLET BY MOUTH~EVERY DAY  ~"
              },
              "timing": {
                "resourceType": "Timing",
                "event": [
                  "1990-01-01T00:00:00"
                ],
                "code": {
                  "coding": [
                    {
                      "system": "chcs",
                      "version": "0.0.1",
                      "code": "chcs",
                      "display": "T1 TAB PO QD #169 RF0",
                      "userSelected": false
                    }
                  ],
                  "text": "T1 TAB PO QD #169 RF0"
                }
              },
              "asNeededBoolean": false
            }
          ]
        },
        {
          "resourceType": "DiagnosticReport",
          "id": "urn:local:fhir:DiagnosticReport:100417-4559064",
          "identifier": [
            [
              {
                "use": "usual",
                "type": {
                  "coding": [
                    {
                      "code": "chcss",
                      "display": "chcss"
                    }
                  ],
                  "text": "chcss:Kg_Patient_Diagnosis-100417"
                },
                "value": "chcss:Kg_Patient_Diagnosis-100417"
              }
            ]
          ],
          "status": "final",
          "category": "OTH",
          "code": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "V70.5 H",
                "userSelected": false
              }
            ],
            "text": "V70.5 H"
          },
          "subject": {
            "reference": "Patient/2-000007",
            "display": "BUNNY,BUGS"
          },
          "conclusion": "V70.5 H",
          "codedDiagnosis": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "V70.5 H",
                "userSelected": false
              }
            ],
            "text": "V70.5 H"
          },
          "presentedForm": [
            {
              "_id": "100417-4559064",
              "type": "chcss:Kg_Patient_Diagnosis-100417",
              "label": "27642;OTHER EXAMINATION DEFINED POPULATION",
              "problem-100417": "27642;OTHER EXAMINATION DEFINED POPULATION",
              "patient-100417": {
                "id": "2-000007",
                "label": "BUNNY,BUGS"
              },
              "status-100417": "Active",
              "location-100417": {
                "id": "44-154",
                "label": "OPTOMETRY ANYHC 1234"
              },
              "date_of_onset-100417": {
                "value": "1990-01-01",
                "type": "xsd:date"
              },
              "diagnosis-100417": "V70.5 H"
            }
          ]
        },
        {
          "resourceType": "DiagnosticReport",
          "id": "urn:local:fhir:DiagnosticReport:100417-4562039",
          "identifier": [
            [
              {
                "use": "usual",
                "type": {
                  "coding": [
                    {
                      "code": "chcss",
                      "display": "chcss"
                    }
                  ],
                  "text": "chcss:Kg_Patient_Diagnosis-100417"
                },
                "value": "chcss:Kg_Patient_Diagnosis-100417"
              }
            ]
          ],
          "status": "final",
          "category": "OTH",
          "code": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "V72.12",
                "userSelected": false
              }
            ],
            "text": "V72.12"
          },
          "subject": {
            "reference": "Patient/2-000007",
            "display": "BUNNY,BUGS"
          },
          "conclusion": "V72.12",
          "codedDiagnosis": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "V72.12",
                "userSelected": false
              }
            ],
            "text": "V72.12"
          },
          "presentedForm": [
            {
              "_id": "100417-4562039",
              "type": "chcss:Kg_Patient_Diagnosis-100417",
              "label": "27224;ENCOUNTER FOR HEARING CONSERVATION AND TREATMENT",
              "problem-100417": "27224;ENCOUNTER FOR HEARING CONSERVATION AND TREATMENT",
              "patient-100417": {
                "id": "2-000007",
                "label": "BUNNY,BUGS"
              },
              "status-100417": "Active",
              "location-100417": {
                "id": "44-5083",
                "label": "HEARING CONSERVATION CLINIC"
              },
              "date_of_onset-100417": {
                "value": "1990-01-01",
                "type": "xsd:date"
              },
              "diagnosis-100417": "V72.12"
            }
          ]
        },
        {
          "resourceType": "DiagnosticReport",
          "id": "urn:local:fhir:DiagnosticReport:100417-4568875",
          "identifier": [
            [
              {
                "use": "usual",
                "type": {
                  "coding": [
                    {
                      "code": "chcss",
                      "display": "chcss"
                    }
                  ],
                  "text": "chcss:Kg_Patient_Diagnosis-100417"
                },
                "value": "chcss:Kg_Patient_Diagnosis-100417"
              }
            ]
          ],
          "status": "final",
          "category": "OTH",
          "code": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "V70.5 2",
                "userSelected": false
              }
            ],
            "text": "V70.5 2"
          },
          "subject": {
            "reference": "Patient/2-000007",
            "display": "BUNNY,BUGS"
          },
          "conclusion": "V70.5 2",
          "codedDiagnosis": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "V70.5 2",
                "userSelected": false
              }
            ],
            "text": "V70.5 2"
          },
          "presentedForm": [
            {
              "_id": "100417-4568875",
              "type": "chcss:Kg_Patient_Diagnosis-100417",
              "label": "19227;PERIODIC PREVENTION EXAMINATION",
              "problem-100417": "19227;PERIODIC PREVENTION EXAMINATION",
              "patient-100417": {
                "id": "2-000007",
                "label": "BUNNY,BUGS"
              },
              "status-100417": "Active",
              "location-100417": {
                "id": "44-38",
                "label": "MILITARY SICKCALL ANYHC 1234"
              },
              "date_of_onset-100417": {
                "value": "1990-01-01",
                "type": "xsd:date"
              },
              "diagnosis-100417": "V70.5 2"
            }
          ]
        },
        {
          "resourceType": "Procedure",
          "id": "urn:local:fhir:Procedure:Procedure-1074046",
          "identifier": [
            {
              "use": "usual",
              "type": {
                "coding": [
                  {
                    "code": "chcss",
                    "display": "chcss"
                  }
                ],
                "text": "1074046"
              },
              "value": "Procedure/1074046"
            }
          ],
          "subject": {
            "reference": "Patient/2-000007",
            "display": "BUNNY, BUGS"
          },
          "status": "completed",
          "category": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "HDDConcept-1450368",
                "userSelected": false
              }
            ],
            "text": "HDDConcept-1450368"
          },
          "code": {
            "coding": [
              {
                "system": "chcs",
                "version": "0.0.1",
                "code": "chcs",
                "display": "HDDConcept-1450368",
                "userSelected": false
              }
            ],
            "text": "HDDConcept-1450368"
          },
          "performedDateTime": "1990-01-01T00:00:00",
          "encounter": {
            "display": "Encounter Procedure"
          }
        }
      ]
    }
  ]
}
